src/interfaces/ISablierV2MerkleLockupFactory.sol

```diff
@@ -37,6 +37,16 @@ interface ISablierV2MerkleLockupFactory {
         uint256 recipientCount
     );
 
+    /*//////////////////////////////////////////////////////////////////////////
+                                 CONSTANT FUNCTIONS
+    //////////////////////////////////////////////////////////////////////////*/
+
+    /// @notice Verify the sum of percentages in `tranches` equals 100%.
+    /// @dev Reverts if the sum of percentages overflows.
+    /// @param tranches The tranches with their respective unlock percentages.
+    /// @return `true` if the sum of percentages equals 100%.
+    function isValidMerkleLT(MerkleLT.TrancheWithPercentage[] calldata tranches) external pure returns (bool);
```

Comment:
let's add named return value, and remove the return natspec, also let's mention the value of 100%:

```solidity
   /// @notice Verify the sum of percentages in `tranches` equals 100% ,i.e. 1e18.
   /// @dev Reverts if the sum of percentages overflows.
   /// @param tranches The tranches with their respective unlock percentages.
   function isValidMerkleLT(MerkleLT.TrancheWithPercentage[] calldata tranches) external pure returns (bool result);
```

Reply:
10f690ededd35f7f0452a20b4ed08f076bf4f6e3